config/retry/config.go

```diff
@@ -96,6 +98,50 @@ func NewConfig(name string, metric *prometheus.Observer, backoffFnCfg *BackoffFn
 	}
 }
 
+type RetryRateLimiter struct {
```

Comment:
Need comments like for exported functions and type definitions, 
```
// RetryRateLimiter is used to limit retry times for PD requests.
```
or the lint check would fail.